index.html

```diff
@@ -66,6 +64,188 @@ <h2>I’am a Software Developer</h2>
           <img src="Icons/side_img.png" alt="" />
         </div>
       </section>
+
+      <section class="work-section">
+        <div class="work-wrap">
+          <h1>My Recent Works</h1>
+          <div class="title-line"></div>
+          <div class="work-hint">
+            <img src="Icons/work_img.png" alt="" />
+
+            <h2>Multi-Post Stories</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div id="tech-lang">
+              <ul>
+                <li>css</li>
+                <li>html</li>
+                <li>bootsrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <button id="hint-button" type="button">See Project</button>
```

Comment:
- [ ] Please the button (e.g., `See project`) follow the button styling: Active, Hover, and Pressed (see design)
- [ ] To be apply in all button
![image](https://user-images.githubusercontent.com/63972670/159126293-3c1ebeab-fce0-4368-8e15-2869fc273bef.png)

Reply:
Thank you